eo-maven-plugin/src/main/java/org/eolang/maven/UnplaceMojo.java

```diff
@@ -82,78 +84,90 @@ public void exec() throws IOException {
                 new Rel(this.placed)
             );
         } else {
-            this.placeThem();
+            this.unplaceBinaries();
+            this.unplaceDependencies();
         }
     }
 
+    /**
+     * Mark dependencies as unplaced if all related binaries are unplaced.
+     */
+    private void unplaceDependencies() {
```

Comment:
@volodya-lombrozo Maybe we can rename `dependencies`/`binaries` into `jars`/`classes`? It's quite hard to guess what is 'dependency' and what is 'binary' in this context.

Reply:
@mximp Done, please, have a look